changelogs/unreleased/3764-fix-deprecation-pytest.yml

```diff
@@ -0,0 +1,6 @@
+description: Add pytest-asyncio mode to remove deprecation warnings
+change-type: patch
+destination-branches: [master, iso4, iso5]
```

Comment:
I don't think we should merge this change on the `iso4` branch. `iso4` is still running on `pytest-asyncio==0.16.0`, because it runs on `python3.6`.